source/ui/BioXAS/BioXASSSRLMonochromatorRegionControlEditor.cpp

```diff
@@ -2,7 +2,7 @@
 #include "beamline/BioXAS/BioXASSSRLMonochromatorRegionControl.h"
 
 BioXASSSRLMonochromatorRegionControlEditor::BioXASSSRLMonochromatorRegionControlEditor(BioXASSSRLMonochromatorRegionControl *regionControl, QWidget *parent) :
-	BioXASControlEditor(regionControl, parent)
+	CLSControlEditor(regionControl, parent)
 {
 
 }
```

Comment:
Line 21 in this file still refers to BIOXASXControlEditor::setControl(newControl);

Should be CLSControlEditor, I believe.